t4c/t4c_cli/util/datetime.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import datetime
+import logging
+
+log = logging.getLogger(__name__)
+
+
+def format_datetime_to_isoformat_without_tz(
+    datetime_arg: datetime.datetime,
+) -> str:
+    """Format the provided datetime object to a string in ISO format without timezone information.
+
+    Returns
+    -------
+    str
+        The formatted datetime string in the format: "yyyy-MM-ddThh-mm-ss.SSS".
+    """
```

Comment:
```
   Returns
    -------
    str
        The formatted datetime string in the format: "YYYY-MM-DDTHH:MM:SS.sss".
        For example, a datetime object representing April 8, 2024, at 15:30:45.124567 would be formatted as "2024-04-08T15:30:45.124".
```

There seems to be a small problem with the format provided in the doc here, i.e. it uses `-` instead of `:` for the time. Also, I think having an example here would be very helpful in understanding what format is returned.